src/bosh-director/lib/bosh/director/jobs/delete_vm.rb

```diff
@@ -17,23 +17,31 @@ def initialize(vm_cid)
 
       def perform
         logger.info("deleting vm: #{@vm_cid}")
-        begin
-          instance = Bosh::Director::Api::InstanceLookup.new.by_vm_cid(@vm_cid).first
-          @deployment_name = instance.deployment.name
-          @instance_name = instance.name
-          parent_id = add_event
-          @vm_deleter.delete_for_instance(instance, false)
-        rescue InstanceNotFound
-          parent_id = add_event
-          @vm_deleter.delete_vm_by_cid(@vm_cid)
+        event_log_stage = Config.event_log.begin_stage("Delete VM", 1)
+        event_log_stage.advance_and_track(@vm_cid, false) do
```

Comment:
@dpb587-pivotal the main idea, as far as I remember, that in case of vm delete, there are 2 exceptions - VMNotFound and InstanceNotFound, and both of them shouldn't be shown in logs as task failure. However today I haven't been able to recreate this misleading behaviour, so let me update the pr, removing this logic